include/class.forms.php

```diff
@@ -3896,28 +3896,27 @@ function render($options) {
             function($t) { return strpos($t, '/') !== false; }
         );
         $maxfilesize = ($config['size'] ?: 1048576) / 1048576;
-        $files = $F = array();
+        $files = array();
```

Comment:
unused variable??

Reply:
Looking at it now, I need to replace it with `$attachments`. what do you think?

Reply:
Nothing to think about - delete it - it's not being used, right?